### PR TITLE
Add configuration file for issue templates

### PR DESCRIPTION
**Pull request by Jarmos-san**
The configuration file for issue templates helps disable create blank issue threads & also will point the issue thread creator to the [developersIndia subreddit](https://www.reddit.com/r/developersIndia) for any QnAs.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```
Comment (Bhupesh-V) [inline .github/ISSUE_TEMPLATE/config.yml:1]: What if the questions are regarding the project, in that case, we would need to have blank templates right?

Comment (Jarmos-san) [reply .github/ISSUE_TEMPLATE/config.yml:1]: For questions it's generally recommended to use [Discussion](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions) threads for better SEO & discoverability on Search Engines. Let me know if it still needs to be changed though?

Comment (Bhupesh-V) [reply .github/ISSUE_TEMPLATE/config.yml:1]: cool makes sense

Comment (Bhupesh-V) [reply .github/ISSUE_TEMPLATE/config.yml:1]: So sometime i report broken links to projects, where will that classify 😅 ?

**Review by Bhupesh-V — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (Jarmos-san) [reply .github/ISSUE_TEMPLATE/config.yml:1]: > So sometime i report broken links to projects, where will that classify 😅 ?

Yeah makes sense. In that case in I'll configure the Bug Report template accordingly if that works? For context, I'll make changes to #4 to consider broken links in any of the documentation, READMEs, etc. Does that work then?

Comment (Bhupesh-V) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Yup 👍🏽

**Pull request merged**